app/serializers/rest/status_serializer.rb

```diff
@@ -13,6 +13,7 @@ class REST::StatusSerializer < ActiveModel::Serializer
   attribute :muted, if: :current_user?
   attribute :bookmarked, if: :current_user?
   attribute :pinned, if: :pinnable?
+  has_many :filter_results, serializer: REST::FilterResultSerializer, if: :current_user?
```

Comment:
How about naming it `filtered`, or `filters`?

Reply:
Renamed to `filtered`.